cpp/dolfinx/fem/FunctionSpace.h

```diff
@@ -82,9 +82,9 @@ class FunctionSpace
   /// `(num_points, gdim)`, otherwise it is transposed and has shape
```

Comment:
I guess the documentation regarding shape is not really correct any longer.
Similarly the return type is no longer correct.